src/arch/xtensa/lib/cpu.c

```diff
@@ -28,11 +28,6 @@
 #include <xtos-structs.h>
 #include <stdint.h>
 
-/* cpu tracing */
```

Comment:
@mmaka1 great change. 

I wonder if we can use trace_err, trace_dbg, trace_info instead of tr_err, tr_dgb, tr_info. It is much more clear.

Reply:
This was intentional and I actually followed the kernel pr_...() pattern used for macros that simplify printk calls, i order to make sure that names are as short as possible and there are more space for the message.

Reply:
Wouldn't `tr_* ` be more intuitive than `buf_dbg`, `buf_err` etc names ? When I saw `buf_dbg` first time I thought it does more than just tracing.

Reply:
> This was intentional and I actually followed the kernel pr_...() pattern used for macros that simplify printk calls, i order to make sure that names are as short as possible and there are more space for the message

Ok, it makes sense.

Reply:
@jajanusz buf_ is one of specific APIs w/ different parameters to retrieve instance ID as well and that is why the names are different. tr_ is similar to pr_ in the kernel while buf_(pipe,comp,dai) are similar to dev_.

Reply:
@mmaka1 is it possible in the feature to have an unified api for buf_(pipe,comp,dai) just like dev_ is in the linux kernel?

Reply:
@dbaluta There isn't a common type between buffers, pipelines, components and DAIs nor a common ID namespace so we need different macros for that.